### ISSUE TITLE
[Bug]: Recorded requests in Herd's dump log are not rewinded

### ISSUE THREAD

**Issue by themazim** [beyondcode/herd-community#1046]
### Platform

macOS

### Operating system version

macOS Sonoma 14.6.1 (23G93)

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.11.0 (Build: 33)

### PHP Version

PHP 8.2 (8.2.2.20)

### Bug description

Implemented API requests for a service and was getting `null` on getContents all the time. yet, when I tried manually in the browser the requests just worked fine.

Turns out the requests were already read by herd for logging and I had to rewind them first. 

I would have expected to get a request which has not been altered / "tinkered" with. A simple rewind after reading the request should fix this in herd. 

### Steps to reproduce

Run any Http Request and try to get its contents while request debugging is enabled:

```
use Illuminate\Support\Facades\Http;

$response = Http::get('https://google.com'); 
dump($response->getBody()->getContents());

```

**Comment by mpociot** [comment]
Right, this seems to happen because we get the response body as well to show it in the dump UI.
This will be fixed in the next release.

**Comment by mpociot** [comment]
This is now fixed in Herd 1.11.1
